### PR TITLE
MODE-2094: Access list support fot CMIS connector

### PR DESCRIPTION
**Pull request by okulikov**
This PR allows to read access lists defined by CMIS repository.

### CODE REVIEW

File: connectors/modeshape-connector-cmis/src/test/java/org/modeshape/connector/cmis/CmisConnectorIT.java [hunk 1]
```diff
@@ -234,7 +236,7 @@ public void shouldAccessCreationDatePropertyForFolder() throws Exception {
         assertTrue(date != null);
     }
 
-    // @Test
+//     @Test
```
Comment (hchiorean) [inline connectors/modeshape-connector-cmis/src/test/java/org/modeshape/connector/cmis/CmisConnectorIT.java]: should this be commented out ?

**Comment by hchiorean** [comment]
@okulikov: with the exception of the question, this looks good to me

**Comment by okulikov** [comment]
@hchiorean , it was commented and actually I don't remember why. When I uncomment this test fails but not due to access list. Do we want to fix this bug in this PR as well?

**Comment by hchiorean** [comment]
sure. If there's a bug somewhere in the connector, we should fix it. You can add a separate commit to this PR for the fix.

**Comment by okulikov** [comment]
@hchiorean , do we want to squash both commits?

**Comment by hchiorean** [comment]
@okulikov: no, I think a separate commit is fine as it underlines that there's a fix for a bug
@rhauch: this looks good to me, are you ok with merging ?

**Pull request merged**